getGitHubContributions.js

```diff
@@ -106,8 +103,20 @@ function getGitHubData() {
             reviewedPR => octokit.paginate(
                 `GET ${reviewedPR.url.slice('https://api.github.com'.length)}/timeline`,
                 {headers: {Accept: 'application/vnd.github.mockingbird-preview'}}
-            )
-        ))
+            )))
+            .then(events => {
```

Comment:
I think that this does basically the same thing https://underscorejs.org/#flatten as this whole block

```suggestion
            .then(events => _.flatten(events))
```

Reply:
Didn't know this was a thing.